cloud/docker_client.go

```diff
@@ -396,8 +402,14 @@ func (c *dockerClientImpl) GetDockerLogs(ctx context.Context, h *host.Host, cont
 		return nil, errors.Wrapf(err, "Error copying stream for container %s", containerID)
 	}
 
-	res.OutReader = tempout
-	res.ErrReader = temperr
+	res.OutStr = tempout.String()
+	res.ErrStr = temperr.String()
```

Comment:
I'm a bit worried about overloading the app servers memory by streaming the whole thing through memory.

Reply:
Is there something that stops us from streaming logs directly from the docker host to the agent?

Reply:
We should do this, though it can be a different ticket